docs/azure-monitor-logging.md

```diff
@@ -0,0 +1,81 @@
+# Azure Monitor tagging
+
+All telemetry sent to Azure Monitor, whether traces, metrics, or spans, should follow this guide.
+
+## Overloading of default dimensions
+
+A few built-in Azure Monitor fields are overloaded:
+
+| field | overload |
+| - | - |
+| `cloud_RoleName` | Either `device` or `service` |
+| `cloud_RoleInstance` | `runId` for the process producing telemetry. |
+
+## customDimensions
+
+```json
+  {
+    "sdkLanguageVersion": "3.8.2",
+    "sdkLanguage": "python",
+    "sdkVersion": "2.2.3",
+    "lineNumber": "291",
```

Comment:
line number? of what? where the log came from? I don't think that is available in .NET.

Reply:
I've now read further down and understand some of these are python-specific and automatically added. I'd suggest removing them from this example, as they aren't part of the spec (what we expect) but rather extra payload you get. Documenting them lower down should be sufficient.